homeassistant/components/camera/__init__.py

```diff
@@ -315,7 +314,8 @@ def __init__(self):
         """Initialize a camera."""
         self.is_streaming = False
         self.content_type = DEFAULT_CONTENT_TYPE
-        self.access_tokens: Deque[str] = collections.deque([], 2)
+        # Cannot properly type the following line until Python 3.6.1
+        self.access_tokens = collections.deque([], 2)  # type: ignore
```

Comment:
I'm not 100% sure if it would be proper typing, but simply annotating it as `collections.deque` appears to at least pass mypy.